docs/en/operations/system_tables.md

```diff
@@ -482,6 +482,76 @@ When the table is deleted manually, it will be automatically created on the fly.
+- `master_os_thread_id` (Int32) — OS initial thread ID.
+- `query` (String) — Query string.
+- `is_initial_query` (UInt8) — Query type. Possible values:
+    - 1 — Query was initiated by the client.
+    - 0 — Query was initiated by another query for distributed query execution.
+- `user` (String) — Name of the user who initiated the current query.
+- `query_id` (String) — ID of the query.
+- `address` (FixedString(16)) — IP address the query was initiated from.
+- `port` (UInt16) — The server port that was used to receive the query.
+- `initial_user` (String) —  Name of the user who ran the parent query (for distributed query execution).
+- `initial_query_id` (String) — ID of the parent query.
+- `initial_address` (FixedString(16)) — IP address that the parent query was launched from.
+- `initial_port` (UInt16) — The server port that was used to receive the parent query from the client.
+- `interface` (UInt8) — Interface that the query was initiated from. Possible values:
+    - 1 — TCP.
+    - 2 — HTTP.
+- `os_user` (String) — User's OS.
+- `client_hostname` (String) — Server name that the [clickhouse-client](../interfaces/cli.md) is connected to.
+- `client_name` (String) — The [clickhouse-client](../interfaces/cli.md) name.
+- `client_revision` (UInt32) — Revision of the [clickhouse-client](../interfaces/cli.md).
+- `client_version_major` (UInt32) — Major version of the [clickhouse-client](../interfaces/cli.md).
+- `client_version_minor` (UInt32) — Minor version of the [clickhouse-client](../interfaces/cli.md).
+- `client_version_patch` (UInt32) — Patch component of the [clickhouse-client](../interfaces/cli.md) version.
+- `http_method` (UInt8) — HTTP method that initiated the query. Possible values:
+    - 0 — The query was launched from the TCP interface.
+    - 1 — `GET` method was used.
+    - 2 — `POST` method was used.
+- `http_user_agent` (String) — The `UserAgent` header passed in the HTTP request.
+- `quota_key` (String) — The quota key specified in the [quotas](quotas.md) setting.
+- `revision` (UInt32) — ClickHouse revision.
+- `ProfileEvents.Names` (Array(String)) — Counters that measure the following metrics:
+    - Time spent on reading and writing over the network.
+    - Time spent on reading and writing to a disk.
+    - Number of network errors.
+    - Time spent on waiting when the network bandwidth is limited.
+- `ProfileEvents.Values` (Array(UInt64)) — Values of metrics that are listed in the&#160;`ProfileEvents.Names` column.
+
+By default, logs are added to the table at intervals of 7.5 seconds. You can set this interval in the [query_thread_log](server_settings/settings.md#server_settings-query-thread-log) server setting (see the `flush_interval_milliseconds` parameter). To flush the logs forcibly from the memory buffer into the table, use the `SYSTEM FLUSH LOGS` query.
+
+When the table is deleted manually, it will be automatically created on the fly. Note that all the previous logs will be deleted.
+
+!!! note
```

Comment:
A link to TTL.

Reply:
Is it possible to alter *_log tables with setting TTL? https://clickhouse.yandex/docs/en/operations/table_engines/mergetree/#table_engine-mergetree-ttl

Or is it possible to set it in the config directly?

Reply:
It's possible to ALTER to add TTL. It's not possible to specify in config directly.